### ISSUE TITLE
Not compatible updated PlotSquared

### ISSUE THREAD

**Issue by andris155** [utarwyn/EnderContainers#18]
```
[11:17:01] [Server thread/ERROR]: Error occurred while enabling EnderContainers v2.1.0 (Is it up to date?)
java.lang.NoClassDefFoundError: com/intellectualcrafters/plot/flag/Flag
	at java.lang.Class.forName0(Native Method) ~[?:?]
	at java.lang.Class.forName(Class.java:315) ~[?:?]
	at fr.utarwyn.endercontainers.dependencies.DependenciesManager.loadDependencies(DependenciesManager.java:102) ~[?:?]
	at fr.utarwyn.endercontainers.dependencies.DependenciesManager.initialize(DependenciesManager.java:50) ~[?:?]
	at fr.utarwyn.endercontainers.AbstractManager.<init>(AbstractManager.java:40) ~[?:?]
	at fr.utarwyn.endercontainers.dependencies.DependenciesManager.<init>(DependenciesManager.java:38) ~[?:?]
	at fr.utarwyn.endercontainers.EnderContainers.onEnable(EnderContainers.java:45) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:254) ~[patched_1.13.2.jar:git-Paper-492]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:326) ~[patched_1.13.2.jar:git-Paper-492]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:405) ~[patched_1.13.2.jar:git-Paper-492]
	at org.bukkit.craftbukkit.v1_13_R2.CraftServer.enablePlugin(CraftServer.java:441) ~[patched_1.13.2.jar:git-Paper-492]
	at org.bukkit.craftbukkit.v1_13_R2.CraftServer.enablePlugins(CraftServer.java:355) ~[patched_1.13.2.jar:git-Paper-492]
	at net.minecraft.server.v1_13_R2.MinecraftServer.l(MinecraftServer.java:607) ~[patched_1.13.2.jar:git-Paper-492]
	at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:569) ~[patched_1.13.2.jar:git-Paper-492]
	at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:430) ~[patched_1.13.2.jar:git-Paper-492]
	at net.minecraft.server.v1_13_R2.DedicatedServer.init(DedicatedServer.java:316) ~[patched_1.13.2.jar:git-Paper-492]
	at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:785) ~[patched_1.13.2.jar:git-Paper-492]
	at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: java.lang.ClassNotFoundException: com.intellectualcrafters.plot.flag.Flag
	at java.net.URLClassLoader.findClass(URLClassLoader.java:471) ~[?:?]
	at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:138) ~[patched_1.13.2.jar:git-Paper-492]
	at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:84) ~[patched_1.13.2.jar:git-Paper-492]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:588) ~[?:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:521) ~[?:?]
	... 18 more
```

**Comment by utarwyn** [comment]
Hello @andris155 

What version of PlotSquared causes this bug? **4.0-pre-release?**
Thank you for the feedback! I will work on it very quickly.

Best regards

**Comment by utarwyn** [comment]
Hello @andris155,

This bug should be fixed with my recent commit 92784e387cf1d83a99ed45f8274175b429a16173 in which I have worked to support the new version of PlotSquared. Please note that in future versions of EnderContainers, only **PlotSquared 4+** will be supported.

Is it good for you? If so, you can close this issue :+1:.

Regards.

**Comment by andris155** [comment]
Works, thanks!
